src/components/DataProvider/index.js

```diff
@@ -131,6 +131,14 @@ export default class DataProvider extends Component {
         this.startUpdateInterval();
       }
     }
+
+    // check if pointsPerSeries changed in props -- if so fetch new data
+    if (!isEqual(this.props.pointsPerSeries, prevProps.pointsPerSeries)) {
+      await Promise.map(this.props.series, s =>
+        this.fetchData(s.id, 'UPDATE_POINTS_PER_SERIES')
```

Comment:
@martincognite do we have a list of reasons maintained anywhere?

Reply:
In `README.md` there's a list.